SoftLayer/CLI/block/subnets/assign.py

```diff
@@ -0,0 +1,37 @@
+"""Assign block storage subnets to the given host id."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('access_id', type=int)
+@click.option('--subnet-id', multiple=True, type=int,
+              help="ID of the subnets to assign; e.g.: --subnet-id 1234")
+@environment.pass_env
+def cli(env, access_id, subnet_id):
+    """Assign block storage subnets to the given host id.
+
+    access_id is the host_id obtained by: slcli block access-list <volume_id>
+    """
+    try:
+        subnet_ids = list(subnet_id)
+        block_manager = SoftLayer.BlockStorageManager(env.client)
+        assigned_subnets = block_manager.assign_subnets_to_acl(access_id, subnet_ids)
+
+        for subnet in assigned_subnets:
+            message = "{0}".format("Successfully assigned subnet id: " + str(subnet) +
```

Comment:
should be this:
```python
click.echo("Successfully assigned subnet id: {} to allowed host id: {}.".format(subnet, access_id))
```

.format() will substitute in order, so for simple strings like this you don't really need to number/name  the substitution brackets. But its certainly fine if you want to be verbose.